inbm/dispatcher-agent/dispatcher/sota/os_updater.py

```diff
@@ -69,7 +70,8 @@ def update_local_source(self, file_path: str) -> List[str]:
         pass
 
     @staticmethod
```

Comment:
Should we remove `@staticmethod` and keep only `@abstractmethod`?

Reply:
@yengliong93 I think it's both--static meaning no self references (including in sub classes), and abstract meaning must be implemented in sub class or the sub class becomes abstract.